release/kata-deploy-binaries.sh

```diff
@@ -112,6 +112,16 @@ install_qemu() {
 	tar xf kata-qemu-static.tar.gz -C "${destdir}"
 }
 
+# Install static firecracker asset
+install_firecracker() {
+	info "build static firecracker"
+	"${script_dir}/../static-build/firecracker/build-static-firecracker.sh"
+	info "Install static firecracker"
```

Comment:
is there a not-static?

Reply:
Its is always built as static. Clearlinux does not build it as static though (libc)

Reply:
ok.  I was just thinking the naming might not need static in it, but was mostly just curious.  looks good...